updateFallbackDirs.py

```diff
@@ -153,8 +153,8 @@
 # When True, they are included, when False, they are excluded
 INCLUDE_UNLISTED_ENTRIES = True if OUTPUT_CANDIDATES else False
 
-WHITELIST_FILE_NAME = 'scripts/maint/fallback.whitelist'
-FALLBACK_FILE_NAME  = 'src/app/config/fallback_dirs.inc'
+WHITELIST_FILE_NAME = 'fallback.whitelist'
+FALLBACK_FILE_NAME  = '../tor/src/app/config/fallback_dirs.inc'
```

Comment:
Should this look at an environment variable or something to override the default?

Reply:
Done in 18a5ef4